tests/system_tests_expandvars.py

```diff
@@ -0,0 +1,91 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import os
+import sys
+import subprocess
```

Comment:
Working like this will make it hard to report test coverage, if we ever wanted to do that... https://coverage.readthedocs.io/en/7.0.2/subprocess.html

Alternative would be to write the minimum number of subprocess tests we need (which is two, for wrong args and for correct args) and then a test that imports `expandvars.py` as a module and calls `expand_file(input_filename, output_filename)` directly. But, that is not worth the effort, probably. And only testing `expand_file` would mean we would not test running the script, which is also important to test.